src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { useSearchParams } from 'react-router-dom';
+import { PeopleList } from '../PeopleList';
+import { SearchLink } from '../SearchLink';
+import { ESortBy } from '../../types';
+
+const tableHeaders: ESortBy[]
+  = [ESortBy.Name, ESortBy.Sex, ESortBy.Born, ESortBy.Died];
+
+export const PeopleTable = () => {
+  const [searchParams] = useSearchParams();
+  const sortParam = searchParams.get('sort') ?? '';
+  const orderParam = searchParams.get('order') ?? '';
+
+  const getSortStyleLink = (param: string) => {
+    if (sortParam === param && orderParam) {
+      return 'fa-sort-down';
+    }
+
+    if (sortParam === param) {
+      return 'fa-sort-up';
+    }
+
+    return 'fa-sort';
+  };
+
+  const sortBy = (sortField: ESortBy) => {
+    if (sortParam !== sortField) {
+      return { sort: sortField, order: null };
+    }
+
+    if (!orderParam) {
+      return { sort: sortField, order: 'desc' };
+    }
+
+    return { sort: null, order: null };
+  };
```

Comment:
It is better to define this functions outside the component so that it is not created every time a new render is performed.